ts/features/zendesk/components/ZendeskSupportComponent.tsx

```diff
@@ -40,10 +39,29 @@ import {
   zendeskConfigSelector,
   zendeskTicketNumberSelector
 } from "../store/reducers";
+import { isReady } from "../../bonus/bpd/model/RemoteValue";
 
 type Props = {
   assistanceForPayment: boolean;
 };
+
+type DisplayedButtons = "None" | "OpenRequest" | "Both";
+
+const getDisplayedButtons = (
+  ticketsNumber: pot.Pot<number, Error>
+): DisplayedButtons =>
+  pot.fold(
+    ticketsNumber,
+    () => "None",
+    () => "None",
+    v => (v > 0 ? "Both" : "OpenRequest"),
+    _ => "OpenRequest",
+    v => (v > 0 ? "Both" : "OpenRequest"),
+    v => (v > 0 ? "Both" : "None"),
```

Comment:
This is because in this case, we can have as response that the user still hasn't opened a ticket, so we should show only the `OpenRequest` CTA while if the user has a ticket we should show both.

If I show the `OpenRequest` CTA and then the response is that the user has a ticket, the user would see first the `OpenRequest` and then both the buttons that were what we want to avoid.